express-notes-api/index.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const app = express();
+
+const fs = require('fs');
+const dataJson = require('./data.json');
+
+let nextID = 10;
```

Comment:
The nextId should only come from the data model you get from `data.json`.  Your app should not always start with a next id of 10. Use `dataJson.nextId` instead.